Services/CaptionService.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Pumukit\YoutubeBundle\Services;
+
+use Pumukit\SchemaBundle\Document\MultimediaObject;
+use Pumukit\YoutubeBundle\Document\Youtube;
+use Pumukit\YoutubeBundle\Document\Caption;
+
+class CaptionService extends YoutubeService
```

Comment:
Why extends `YoutubeService`?

Reply:
@rubenrua To reuse some of its functions as sending email, get youtube document, etc.
In the future, YoutubeService could be the base and each functionality have its own Service (for videos, playlists, captions, etc.) extending from YoutubeService.